examples/04_fuzzy_joining.py

```diff
@@ -416,12 +396,16 @@
 from sklearn.model_selection import GridSearchCV
 
 # We will test four possible values of match_score:
-params = {"joiner__match_score": [0.2, 0.3, 0.4, 0.5]}
+params = {
+    "joiner-1__match_score": [0.2, 0.9],
+    "joiner-2__match_score": [0.2, 0.9],
+    "joiner-3__match_score": [0.2, 0.9],
+}
 
 grid = GridSearchCV(pipeline, param_grid=params)
 grid.fit(df, y)
 
-print(grid.best_params_)
+print(grid.cv_results_)
```

Comment:
I think that the following will render better on the web page, as sphinx gallery will pretty print the list.
```suggestion
grid.cv_results_
```

Reply:
oops it is actuall the best_params_ we want; I removed the print as well